docs/setup/releases/update-strategies: document airgapped updates (#211)
flatcar-linux/flatcar-docs: docs/setup/releases/update-strategies.md

                                  Attr: priority=1 tries=0 successful=1
```

## Airgapped updates

Updating a machine without Internet access is done in two steps.

> First the wanted update payload is downloaded from a non-airgapped machine, then you copy it to your airgapped machine and run the `flatcar-update` tool.

Reviewer: ```suggestion
First, you need to download the update payload on a non-airgapped machine, then you copy it to your airgapped machine and run the `flatcar-update` tool.
```

> If the `flatcar-update` tool is missing on your machine, [download](https://raw.githubusercontent.com/flatcar-linux/flatcar-docs/main/update-to-flatcar.sh) it first, too.

Reviewer: Is this a correct link? It seems to point to the script that updates coreos to flatcar, not the flatcar update one.

Author: Oh, right
